src/components/InventoryTable/EntityTableToolbar.js

```diff
@@ -83,10 +83,16 @@ const EntityTableToolbar = ({
     const allTagsLoaded = useSelector(({ entities: { allTagsLoaded } }) => allTagsLoaded);
     const allTags = useSelector(({ entities: { allTags } }) => allTags);
     const additionalTagsCount = useSelector(({ entities: { additionalTagsCount } }) => additionalTagsCount);
+    const operatingSystems = useSelector(({ entities: { operatingSystems } }) => operatingSystems);
+    const operatingSystemsLoaded = useSelector(({ entities: { operatingSystemsLoaded } }) => operatingSystemsLoaded);
```

Comment:
Nitpick: Let's make it apparent that this is a boolean by renaming it to 'areOperatingSystemsLoaded'. if you feel this is a long,

Reply:
@mkholjuraev, perhaps we could leave it as it is? in my opinion, the adjective `loaded` is already pointing to the true or false nature of this variable